spec/models/recipe_spec.rb

```diff
@@ -8,6 +8,7 @@
     it { is_expected.to have_db_column :description}
     it { is_expected.to have_db_column :ingredients}
     it { is_expected.to have_db_column :directions}
+    it { is_expected.to have_db_column :user_id}
```

Comment:
Please add specs for associations using ShouldaMatcher

Reply:
Done